CMakeExt/MPI.cmake

```diff
@@ -30,9 +30,18 @@ if (MPI_INCLUDE_PATH AND MPI_LIBRARY)
   elseif ("${MPI_INCLUDE_PATH}" MATCHES "openmpi")
```

Comment:
Thought about that as well, yes. Sometimes I regret touching things in the first place though :D

Will look into your comments.